<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeUserDialog</class>
 <widget class="QWidget" name="ChangeUserDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>184</width>
    <height>146</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Сменить пользователя</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>1</number>
   </property>
   <property name="spacing">
    <number>1</number>
   </property>
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="maximumSize">
      <size>
       <width>182</width>
       <height>144</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="labelUserName">
           <property name="text">
            <string>Имя пользователя:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditLogin">
           <property name="text">
            <string>dosh</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="labelPassword">
           <property name="text">
            <string>Пароль</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditPassword">
           <property name="text">
            <string>1</string>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="pushButtonOk">
           <property name="text">
            <string>Ok</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonCancel">
           <property name="text">
            <string>Отмена</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButtonLogout">
         <property name="text">
          <string>Выйти</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>lineEditPassword</sender>
   <signal>returnPressed()</signal>
   <receiver>pushButtonOk</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>44</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>41</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
